src/components/flat-table/flat-table.component.js

```diff
@@ -4,11 +4,15 @@ import { StyledFlatTableWrapper, StyledFlatTable } from './flat-table.style';
 
 const FlatTable = ({
   children,
-  hasStickyHead
+  hasStickyHead,
+  colorTheme,
+  headerBackground
```

Comment:
I still think this is a `styleOverride` and we can achieve the same thing using backround-color: inherit` when the `colorTheme` is `transparent`